services/resources-getter.js

```diff
@@ -55,15 +55,21 @@ function ResourcesGetter(model, opts, params) {
 
   function getRecords() {
     return model
-      .findAndCountAll({
+      .findAll({
```

Comment:
I am forced to dissociate the objects retrieval and the count for the search.
Without this change, the count generate a bad SQL request that raise an error; it doesn't include joined tables properly.